spyder_kernels/console/kernel.py

```diff
@@ -507,3 +508,12 @@ def _load_wurlitzer(self):
                 get_ipython().run_line_magic('reload_ext', 'wurlitzer')
             except Exception:
                 pass
+    def set_equations_color(self, background_color):
+        try:
+            from sympy import init_printing
+            if background_color=="dark":
+                init_printing(forecolor='White')
+            else:
+                init_printing(forecolor='Black')
+        except:
+            pass
```

Comment:
Should we emit some warning that the method "failed" or something @ccordoba12 ?

Reply:
I don't want to report an error if Sympy is not installed, but excepting on ImportError sometimes picks up other nasty errors with the package dependencies.

So what do you propose?

Reply:
We could check if it is not an import error (use except Exception) and if it is not emit some warning with whatever the error was?